tests/components/devolo_home_control/conftest.py

```diff
@@ -0,0 +1,33 @@
+"""Fixtures for tests."""
+
+import pytest
+
+from tests.async_mock import patch
```

Comment:
Please rebase on latest dev branch and import patch from unittest.mock.

Reply:
Done, also changed it in contest.py.